_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+* [Trino Migration at ARM Treasure Data]({% post_url 2020-07-06-presto-summit-arm-td %})
+* [Trino at Zuora]({% post_url 2020-06-16-presto-summit-zuora %})
+
+Another series of training classes with the project founders was hugely
+successful. It includes very valuable content for any Trino user, from beginners
+to experts, that you should not miss:
+
+* [Advanced SQL in Trino with David]({% post_url 2020-07-15-training-advanced-sql %})
+* [Understanding and Tuning Trino Query Processing with Martin]({% post_url 2020-07-30-training-query-tuning %})
+* [Securing Trino with Dain]({% post_url 2020-08-13-training-security %})
+* [Configuring and Tuning Trino with Dain]({% post_url 2020-08-27-training-performance %})
+
+## Features and code
+
+While everything mentioned is already exciting, the true work is visible in the
+new features and improvements in Trino. It is a long list, but read on. You
+won’t want to miss anything.
+
+### Improvements to ANSI SQL support
+
+A core feature of Trino is the ability to use the same standard SQL for any
+connected data source. These improvements empower all users.
+
+* Variable-precision temporal types, with precision down to picoseconds. This a
+  very important feature for any time critical systems such as financial
+  transactions processing
+* Correct, and now SQL spec compliant Timestamp semantics, making migration of
+  SQL statements from other compliant systems such as many RDBMSs easier
+* Implicit coercions for `INSERT` clause.
+* Support for `RANGE` and `GROUPS`-based window frames
+* More support for various shapes of correlated subqueries
+* Support for `INTERSECT ALL` and `EXCEPT ALL`
+* Experimental support for [recursive queries](/docs/current/sql/select.html?highlight=recursive#with-recursive-clause)
+* Enforcement of `NOT NULL` constraints when inserting data
+* Quantified comparisons (e.g., > ALL (...)) in aggregation
+
+### Other query improvements
+
+A number of other features were added to make querying your data sources with
+Trino even more powerful:
+
+* [T-digest data type](/docs/current/language/types.html#t-digest) and functions
+  for approximate quantile computations
+* Support for setting and reading column comments
+* Numerous new functions including `concat_ws()`, `regexp_count()`,
+  `regexp_position()`, `contains_sequence()`, `murmur3()`,
+  `from_unixtime_nanos()`, `from_iso8601_timestamp_nanos()`,
+  `human_readable_seconds()`, `bitwise` operations, `luhn_check()`,
+  `approx_most_frequent()`, `translate()`, `starts_with()`
+
+## Performance
+
+Trino is already [ludicrously fast](/index.html). But then again, even faster is
+better, so we worked on that:
+
+* Improved pushdown of complex operations into connectors, including
+  [aggregation and topN pushdown](/docs/current/optimizer/pushdown.html). We’ve
+  extended several connectors with these new capabilities.
+* [Dynamic filtering and partition pruning]({% post_url
```

Comment:
We could expand Dynamic filtering. It's a pretty big distinguisher for Trino. There were plenty of phases to this and several additions after, including the update to the API to enable remote DF and add the option to have it in all the connectors. etc...

Reply:
I think it already links to the blog posts and such .. its fine imho